Sources/Rendering/Misc/SynchronizableRenderWindow/vtkObjectManager.js

```diff
@@ -290,8 +289,8 @@ function polydataUpdater(instance, state, context) {
       .then(
         (values) => {
           const array = vtkDataArray.newInstance(Object.assign({ values }, arrayMetadata));
-          // console.log('array', array.getName());
-          arraysToBind.push([instance.get(arrayMetadata.location)[arrayMetadata.location].addArray, [array]]);
+          const regMethod = arrayMetadata.registrationMethod ? arrayMetadata.registrationMethod : 'addArray';
```

Comment:
If you change the dataset reader, you will have to fix the data as well and the python script that `may` generate that attribute.